setup.py

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env python
 
 from setuptools import setup, find_packages
-
+import os
+os.system("git clone 'https://github.com/libindic/pypdflib.git'")
```

Comment:
this is not a good way to handle dependencies

pypdf lib is a dependency for this module. An ideal scenario will be pip having the latest version. We might have added git url because there are some un-released changes which we need for this plugin. Since it is not yet released, we should give git repo as dependency. There are several ways to do this. If `install_requires` is not working, you can use `dependency_links`